cql3/expr/expression.cc

```diff
@@ -212,7 +205,7 @@ get_value(const subscript& s, const evaluation_inputs& inputs) {
         }
         return managed_bytes_opt(data_list[key_int].serialize_nonnull());
     } else {
-        throw exceptions::invalid_request_exception(format("subscripting non-map, non-list column {}", cdef->name_as_text()));
+        throw exceptions::invalid_request_exception(format("subscripting non-map, non-list column {}", s.val));
     }
```

Comment:
Printing in error messages should use non-debug printing using `expression::printer`

Reply:
I filed #12702 because I refuse to use expression::printer.

Reply:
Fixed (using {:user}).